packages/safe-apps-sdk/src/safe/index.ts

```diff
@@ -28,6 +31,96 @@ class Safe {
 
     return response.data;
   }
+
+  async calculateSafeMessageHash(messageBytes: string): Promise<string> {
```

Comment:
@rmeissner I wonder if this method is needed at all. Do you have a use case in mind for it?

Reply:
This should be used with `isMessageSigned`, right?

```js
const message = "I'm the owner of wallet 0x000000";
const messageHash = await sdk.txs.calculateSafeMessageHash(message);
const signed = await sdk.txs.isMessageSigned(messageHash);
```

Reply:
This method calculates the hash, which is a mapping key inside the contract: https://github.com/gnosis/safe-contracts/blob/main/contracts/libraries/SignMessage.sol#L29

It doesn't seem to be needed because the validation method does it, too, so you only need to pass the bytes/bytes32 message.